src/olympia/lib/settings_base.py

```diff
@@ -1698,13 +1698,18 @@ def get_redis_settings(uri):
         'rest_framework.parsers.FormParser',
         'olympia.api.parsers.MultiPartParser',
     ),
+
+    'ALLOWED_VERSIONS': ['v3', 'v4'],
+    'DEFAULT_VERSION': 'v4',
```

Comment:
This has to be documented in the API docs and needs docs about how to switch back and forth. On the other hand, there's no generic `/api/` yet as this points to our legacy API anyway.

Reply:
The docs still need updating.  (And archiving the v3 ones).  `DEFAULT_VERSION` is used by `NamespaceVersioning` internally iirc.